### ISSUE TITLE
Search text not removed - when click the result which already an active url

### ISSUE THREAD

**Issue by Nilanth** [praveenn77/docusaurus-lunr-search#13]
Hi,
The search text in the search bar is not removed once we click the search result.

**Comment by lelouch77** [comment]
Thanks for reporting @Nilanth, I will take a look :+1:

**Comment by lelouch77** [comment]
This issue happens only when you select a result which is already an active URL..

**Comment by Nilanth** [comment]
No, it occurs on all pages!
You can check in this demo itself https://lelouch77.github.io/docusaurus-lunr-search-demo/

**Comment by mattcorner** [comment]
When moving from landing page to documentation the search is cleared. However @Nilanth is right that once in the documentation, the search is never cleared.

**Comment by lelouch77** [comment]
Thats the old documentation link.. try this https://lelouch77.github.io/docusaurus-lunr-search-multilang @Nilanth @mattcorner

**Comment by mattcorner** [comment]
Its a bit inconsistent. Sometimes it does remove clear the search, but a lot of the time it doesn't. Here it is not clearing on the new demo @lelouch77 

This only occurs for doc to doc navigation

![docs-lunr-search-not-clearing](https://user-images.githubusercontent.com/27866636/92212631-2b71d780-ee8a-11ea-8521-39f1ca151446.gif)

**Comment by felipeog** [comment]
> Hi,
> The search text in the search bar is not removed once we click the search result.



On `/src/theme/SearchBar/index.js`, I added `_input.setVal('')` and `_event.target.blur()` on the `handleSelected` function (line 25):

```javascript
const initAlgolia = (searchDocs, searchIndex, DocSearch) => {
      new DocSearch({
        searchDocs,
        searchIndex,
        inputSelector: "#search_input_react",
        handleSelected: (_input, _event, suggestion) => {
          const url = baseUrl + suggestion.url;
          const a = document.createElement("a");
          a.href = url;

          _input.setVal(''); // clear value
          _event.target.blur(); // remove focus
          history.push(url);
        }
      });
  };
```

So the input is cleared and its focus is removed. Hope it helps!

**Comment by praveenn77** [comment]
fixed in new version `3.2.0`
